src/Widgets/TerminalWidget.vala

```diff
@@ -784,8 +883,9 @@ namespace Terminal {
             }
         }
 
-        private string? get_link (Gdk.Event event) {
-            return this.match_check_event (event, null);
+        private string? get_link (double? x, double? y) {
+            int tag = 0;
+            return check_match_at (x, y, out tag);
```

Comment:
```suggestion
            return check_match_at (x, y, null);
```

Reply:
This didn't used to work in Vala but I guess later versions have been fixed - thanks.